app/Http/Controllers/Admin/EmployerController.php

```diff
@@ -99,23 +101,26 @@ public function update(Request $request, $slug)
     public function destroy($slug)
     {
         try {
+            Gate::authorize('crudUser');
             $user = User::where('slug', $slug)->first();
             $employer = Employer::where('user_id', $user->id)->first();
-            unlink("assets/images/employers/" . $employer->logo);
+
+            if(!str_starts_with($user->image , 'default')) {
```

Comment:
the employer has two pics, one coming from user and the logo note that unlink() directly can cause bugs, it doesn't have large impact as we use try catch but still can cause bugs
deleteFile() function provide safe delete it first check if the file exists